metrics/utils.go

```diff
@@ -0,0 +1,55 @@
+package metrics
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var (
+	operator_namespace string
+	hub_pod_name       string
+)
+
+func init() {
+	hub_pod_name = getHubFullName()
+}
+
+func getHubFullName() string {
```

Comment:
Is this required only for this test ? 
If it might be used by may other tests as well then we might want to consider putting at a global level .

Reply:
@sshveta currently it is required for the metric tests only.